app/views/sections/show.html.erb

```diff
@@ -23,6 +23,11 @@
     <li>
       <% if LessonSection.find_by(lesson: lesson, section: @section).homework? %>
         <i class="glyphicon glyphicon-book"></i>
+      <% else %>
+        <i class="glyphicon glyphicon-pencil"></i>
+      <% end %>
+      <% if LessonSection.find_by(lesson: lesson, section: @section).try(:day_of_week) %>
+        <%= LessonSection.find_by(lesson: lesson, section: @section).day_of_week.titlecase %>
```

Comment:
How about replace 29 and 30 with:

```ruby
<%= LessonSection.find_by(lesson: lesson, section: @section).try(:day_of_week).try(:titlecase) %>
```

Also, shouldn't day of week and work type have a presence validation? If so, then we should just add that and remove the `try`s.